resources/views/forum/topics/_post.blade.php

```diff
@@ -19,6 +19,13 @@
     }
 
     $user = $post->userNormalized();
+    $group = isset($topic) && $user->isModeratorForForum($topic->forum)
```

Comment:
should also explicitly pass `group` into the `@include('forum.topics._post_info', ...` part